client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/index.tsx

```diff
@@ -44,14 +45,31 @@ const Launchpad: Step = ( { navigation, flow }: LaunchpadProps ) => {
 		[]
 	);
 
-	if ( ! isLoggedIn ) {
-		window.location.replace( `/home/${ siteSlug }` );
+	if (
+		! isLoggedIn ||
+		launchpadScreenOption === 'off' ||
+		areLaunchpadTasksCompleted( flow ) ||
+		( launchpadScreenOption === false && 'videopress' !== flow )
+	) {
```

Comment:
Does anyone know why that videopress exception was originally added? I wonder if we could get rid of that now. We also do not set launchpad_screen to false any more. Would be nice to clean that line out.